packages/backend/src/database/entities/organizationMemberships.ts

```diff
@@ -8,15 +8,23 @@ export type DbOrganizationMembership = {
     user_id: number;
     created_at: Date;
     role: OrganizationMemberRole;
+    role_uuid?: string | null;
```

Comment:
@stevelikesmusic I created this branch from your migration branch, not the mode.s, it is possible some of this code is already duplicated.